manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml

```diff
@@ -645,7 +645,7 @@ spec:
           - extensions.istio.io
           - telemetry.istio.io
           - gateway.networking.k8s.io
-          - inference.networking.x-k8s.io
+          - inference.networking.k8s.io
```

Comment:
If x-k8s.io is not supported in any production release downstream, this is OK. If we had this in any 2.x release prior, we have to keep the old one and add the new one.